release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+        // .. more code
+   });
+```
+
+Thanks to @Azhovan! :bow: :tada:
+
+### Collect usage data on imported internal modules and outputs [#3525](https://github.com/grafana/k6/pull/3525)
+
+Added the data collection of the used modules and outputs when the [usage report](https://grafana.com/docs/k6/latest/misc/usage-collection) is enabled. The data collection is only related to the built-in k6 modules and outputs. Private, custom modules and extensions [are never collected](https://github.com/grafana/k6/blob/f35e67902605877ebf2c5e9c8673cd7faf4cdc1e/cmd/report.go#L33-L57). The usage report is enabled by default in k6, but it is possible to opt-out using the [no-usage-report](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#no-usage-report) option.
+
+We always want to improve the product, but at the same time we need to pay attention where we allocate our resources. Having data of what are the most used modules and outputs give us a better confidence to make decisions because we are supported by data.
+It will let us know what is the percentage of our users that will benefit from the introduction of a new feature and on the other side, in case of a breaking change, how many of them would be impacted.
+
+## UX improvements and enhancements
+
+- [#3529](https://github.com/grafana/k6/pull/3529) Actually enable by default the k6 cloud traces output.
+- [#3440](https://github.com/grafana/k6/pull/3440) adds a fallback for using built-in certificates if the OS provides none. Thanks to `@mem` for working on it!
+- [browser#1104](https://github.com/grafana/xk6-browser/pull/1104) adds support for browser module traces metadata. Users can define *key-value* metadata that will be included as attributes in every generated span.
+- [browser#1135](https://github.com/grafana/xk6-browser/pull/1135) improves the array output from `console` in the k6 logs.
+- [browser#1137](https://github.com/grafana/xk6-browser/pull/1137), [browser#1145](https://github.com/grafana/xk6-browser/pull/1145) improves the error messages displayed when Chrome or Chromium isn't found.
+- [#3543](https://github.com/grafana/k6/pull/3543) switches documentation URLs to the `grafana.com/docs/k6/latest/`.
+
+## Bug fixes
+
+- [#3485](https://github.com/grafana/k6/pull/3485) fixes the rest API always logging it returned 200, which was found as part of fixing lint issues in the code.
+- [browser#1129](https://github.com/grafana/xk6-browser/pull/1129) mitigates the risk of panics when the website under test uses the `console`.
+- [browser#1133](https://github.com/grafana/xk6-browser/pull/1133) fixes `BigInt` parsing.
+- [browser#1108](https://github.com/grafana/xk6-browser/pull/1108), [browser#1110](https://github.com/grafana/xk6-browser/pull/1110) fixes `isVisible` and `isHidden` so that it does not wait for an element to match with the given `selector`, allowing it to continue on with the test script when elements are not on the page.
```

Comment:
```suggestion
- [browser#1108](https://github.com/grafana/xk6-browser/pull/1108), [browser#1110](https://github.com/grafana/xk6-browser/pull/1110) fixes `isVisible` and `isHidden` so that it doesn't wait for an element to match with the given `selector`, allowing it to continue on with the test script when elements are not on the page.
```